lldb/test/API/lang/swift/clangimporter/objcmain_conflicting_dylibs_bridging_headers/TestSwiftObjCMainConflictingDylibsBridgingHeader.py

```diff
@@ -43,6 +43,7 @@ def test(self):
             'break here', lldb.SBFileSpec('Foo.swift'))
         process.Continue()
         self.expect("fr var foo", "expected result", substrs=["23"])
-        self.expect("expression foo", "expected result", substrs=["$R3", "23"])
-        self.expect("expression $R3", "expected result", substrs=["23"])
-        self.expect("expression $R4", "expected result", substrs=["23"])
+        # FIXME: This should work with precise compiler invocations.
+        #self.expect("expression foo", "expected result", substrs=["$R3", "23"])
+        #self.expect("expression $R3", "expected result", substrs=["23"])
+        #self.expect("expression $R4", "expected result", substrs=["23"])
```

Comment:
same question, should this remain commented out?

Reply:
This looks like a regression, but in reality the output here is random based on the order in which modules are visited. The right thing to do is to enable this only when precise compiler invocations are on, but the decorator isn't powerful enough to detect this. I'll need to follow-up here.